motoman_driver/src/industrial_robot_client/joint_trajectory_action.cpp

```diff
@@ -63,11 +64,7 @@ JointTrajectoryAction::JointTrajectoryAction() :
   // A parameter named topics_list contains the respective information for each of the groups
 
   std::string value;
-  if(!ros::param::search("topics_list", value))
```

Comment:
This error message helps people when the "topics_list" parameter cannot be found.  Was it removed for a reason?